pkg/util/goroutinemap/common/util.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package common contains helper methods and common functionality used by
+// all RoroutineMap implementations.
+package common
+
+import (
+	"fmt"
+	"runtime"
+)
+
+func RecoverFromPanic(operationName string, err *error) {
```

Comment:
It sets the error so that a common error handling routine can be used to handle both errors and panics. I moved it to `pkg/util/runtime` and deleted this file.